CONTRIBUTING.md

```
@@ -57,30 +85,95 @@ Make sure you have read the [Documentation for Setting up the Project](https://g
+
+         1. You can also see your code coverage online for your fork of the repo. This is provided by `codecov.io`
+
+            1. Go to this link: `https://app.codecov.io/gh/XXXX/YYYY` where XXXX is your GitHub account username and YYYY is the name of the repository
+            2. Login to `codecov.io` using your GitHub account, and add your **repo** and **branches** to the `codecov.io` dashboard.
+               ![Debugging Test Demo](/public/images/codecov/authorise-codecov-github.jpg)
+            3. Remember to add the `Repository Upload Token` for your forked repo. This can be found under `Settings` of your `codecov.io` account.
+
+            4. Click on Setup Repo option
+               ![Debugging Test Demo](</public/images/codecov/homescrenn%20(1).jpg>)
+            5. Use the value of this token to create a secret named CODE_COV for your forked repo.
+               [![Code-cov-token.jpg](/public/images/codecov/Code-cov-token.jpg)]()
```

Markdownlint (0.35.0)

No empty links

(MD042, no-empty-links)